openedx_webhooks/test/test_is_valid_payload.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from hashlib import sha1
+import hmac
+import json
+
+import pytest
+
+from openedx_webhooks.utils import is_valid_payload
+
+
+def _signature(secret, payload):
+    return (
+        'sha1=' +
+        hmac.new(str(secret), msg=payload, digestmod=sha1).hexdigest()
+    )
+
+
+@pytest.fixture
+def secret1():
+    return 'top secret'
+
+
+@pytest.fixture
+def secret2():
+    return 'not so top secret'
+
+
+@pytest.fixture
+def payload():
+    return json.dumps('payload')
+
+
+@pytest.fixture
+def good_signature(secret1, payload):
+    return _signature(secret1, payload)
+
+
+@pytest.fixture
+def bad_signature(secret2, payload):
+    return _signature(secret2, payload)
```

Comment:
This isn't a bad signature. It's just not the signature you expect in a test further down. This fixture is only used in that test. It'd be easier to understand the test if you move this into that test function, and call it wrong_signature instead.